test/symforce_function_codegen_test_data/sympy/custom_function_replacement_header.h

```diff
@@ -0,0 +1,12 @@
+// NOTE: this file is a test import for "custom_function_replacement.h" and is NOT autogenerated
```

Comment:
You'll need the license header on this file as well - is it possible to put this outside the `symforce_function_codegen_test_data` folder?  It's nice to be able to assume everything in there is generated

Reply:
I was going to ask you about exactly this, but I'll move it elsewhere in that case

Reply:
What do you think of just adding a `test/include` folder and putting it there?

Reply:
I agree that it's somewhat unclear but I think it will become a little convoluted w.r.t. building

Reply:
I think you should be able to e.g. put it in a `test/include` directory and add that as an include path for either just the tests that use it or all the tests with `target_include_directories` in cmake, and it should just work?

Reply:
We already have quite a few include directories we add like this in `test/CMakeLists.txt`